openedx/features/learner_profile/toggles.py

```diff
@@ -1,29 +0,0 @@
-"""
-Toggles for Learner Profile page.
-"""
-
-
-from edx_toggles.toggles import WaffleFlag
-from openedx.core.djangoapps.site_configuration import helpers as configuration_helpers
-
-# Namespace for learner profile waffle flags.
-WAFFLE_FLAG_NAMESPACE = 'learner_profile'
-
-# Waffle flag to redirect to another learner profile experience.
-# .. toggle_name: learner_profile.redirect_to_microfrontend
-# .. toggle_implementation: WaffleFlag
-# .. toggle_default: False
-# .. toggle_description: Supports staged rollout of a new micro-frontend-based implementation of the profile page.
-# .. toggle_use_cases: temporary, open_edx
-# .. toggle_creation_date: 2019-02-19
-# .. toggle_target_removal_date: 2020-12-31
-# .. toggle_warning: Also set settings.PROFILE_MICROFRONTEND_URL and site's ENABLE_PROFILE_MICROFRONTEND.
-# .. toggle_tickets: DEPR-17
-REDIRECT_TO_PROFILE_MICROFRONTEND = WaffleFlag(f'{WAFFLE_FLAG_NAMESPACE}.redirect_to_microfrontend', __name__)
```

Comment:
Do we need to add any additional tickets to clean up the config for this flag in the LMS? (https://internal.courses.edx.org/admin/waffle/flag/42/change/?_changelist_filters=q%3DREDIRECT)

Reply:
yeah, I'll do that.